cna/docs/sponsoring.md

```diff
@@ -0,0 +1,39 @@
+# Become a Friend of Cloud Native Austria
+
+**Are you a company looking to contribute to the Austrian cloud-native community?**
+
+Join us as a **Friend of Cloud Native Austria** and help support the growth and development of cloud-native technologies and practices across Austria. 
+Your commitment as a sponsor will enable us to continue providing valuable resources, networking opportunities, and educational events tailored to professionals and enthusiasts in the cloud-native ecosystem.
+
+## Sponsorship Contribution
+
+- **Yearly sponsorship of €1,000**
+
+## Sponsorship Benefits
+
+As a Friend of Cloud Native Austria, you will enjoy a range of benefits, including:
+
+- **Visibility on Our Website:**
+Your company logo and a link to your website will be featured prominently on our site, recognizing you as a key supporter of the cloud-native community in Austria.
+- **Event Recognition:**
+We will mention your company at each Cloud Native Austria event, highlighting your support and contribution as a Friend of Cloud Native Austria. 
+This includes all workshops, meetups, and conferences we organize or participate in throughout the year.
+- **Social Media Engagement:**
+We will mention your company in a quarterly social media post across all our platforms including LinkedIn, Twitter, and Facebook. 
```

Comment:
quarterly? or is twice a year fine as well? Just thinking about, that we can always do more but not less we promise.

Maybe we can write something like:
"minimum of 2 social media posts the company is mentioned..." or similar.